packages/web-components/fast-foundation/src/tree-view/README.md

```diff
@@ -6,15 +6,22 @@ custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-comp
 ---
 
 An implementation of a [tree](https://w3c.github.io/aria/#tree) as a web-component.
+The `fast-tree-view` supports other elements being included in the tree structure with the introduction of `slot="item"`.
 
 ## Usage
 
 ```html live
 <fast-design-system-provider use-defaults>
     <fast-tree-view>
         Root
-        <fast-tree-item slot="item">Item 1</fast-tree-item>
-        <fast-tree-item slot="item">Item 2</fast-tree-item>
+        <fast-tree-item>
+            Item 1
+            <fast-divider slot="item"></fast-divider>
```

Comment:
I thought things like `<fast-divider>` would only occur as children of the `<fast-tree-view>` and not `<fast-tree-item>`?